templates/kubernetes/index.html

```diff
@@ -363,7 +363,7 @@ <h2>
                 }}
               </div>
               <div class="p-logo-section__item">
-                {{ image(url="https://assets.ubuntu.com/v1/d0e0b382-spectro-cloud-logo.png",
+                {{ image(url="https://assets.ubuntu.com/v1/19c3c265-spectrocloud-logo.png",
                                 alt="Spectro Cloud",
                                 width="72",
```

Comment:
The aspect ratio has changed so the new width is `87`

```suggestion
                                width="87",
```